src/detectors/RPOTS/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ plugin_add_dd4hep(${PLUGIN_NAME})
 plugin_add_event_model(${PLUGIN_NAME})
 
 # Add include directories (works same as target_include_directories)
-#plugin_include_directories(${PLUGIN_NAME} SYSTEM PUBLIC ${podio_INCLUDE_DIR} ${EDM4EIC_INCLUDE_DIR})
+plugin_include_directories(${PLUGIN_NAME} SYSTEM PUBLIC ${podio_INCLUDE_DIR} ${EDM4EIC_INCLUDE_DIR})
 
 # Add libraries (works same as target_include_directories)
-#plugin_link_libraries(${PLUGIN_NAME} EDM4HEP::edm4hep )
+plugin_link_libraries(${PLUGIN_NAME} EDM4HEP::edm4hep )
```

Comment:
I wonder why `plugin_add_event_model` call above is not sufficient. It does the same:
https://github.com/eic/EICrecon/blob/f537c7644b6f8a808ec3687a00b01b635ae93ae7/cmake/jana_plugin.cmake#L209-L234

Reply:
I am not sure about this part - perhaps someone else can commtn.

Reply:
Okay, I made this change, and everything still worked. Learned something new - thanks, Dima.